core/templates/pages/exploration-editor-page/services/exploration-data.service.ts

```diff
@@ -149,7 +150,7 @@ export class ExplorationDataService {
           .fetchApplyDraftExplorationAsync(
             this.explorationId).then((response) => {
             this.loggerService.info('Retrieved exploration data.');
-            this.loggerService.info(JSON.stringify(response));
+            this.loggerService.info('Success');
```

Comment:
Here and below, it should be OK to remove the logging statements.

Reply:
> Here and below, it should be OK to remove the logging statements.

@kevintab95 i did not removed it because it is used in frontend test for knowing that data is successfully fetched or not.

Reply:
We'll need to update the frontend test such that it checks "this.data" is updated with the response correctly. We should not be using logs in the codebase for the sake of tests.

Reply:
Ok @kevintab95 i will update tests for this.

Reply:
@kevintab95 i have updated the frontend tests for this and now it is workng fine.

Reply:
@kevintab95 PTAL